charts/victoria-logs-single/templates/extra-outputs.yaml

```diff
@@ -5,13 +5,16 @@
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  name: victorialogs-outputs
+  name: vl-outputs
   namespace: {{ include "vm.namespace" . }}
   labels: {{- include "vm.labels" $ctx | nindent 4 }}
 data:
-  output_vlogs.conf: |
-    {{- range $i := until (int .Values.server.replicaCount) }}
+  output_vl.conf: |
+    {{- $replicaCount := ternary (int .Values.server.replicaCount) 1 .Values.server.statefulSet.enabled }}
```

Comment:
In statefulsete mode: each victorialogs instance stores all data, and during query, we can get complete data from any instance;
In deployment mode: data is divided among multiple victorialogs instances due to a single write entrance. In order to get complete data, we must query all the victorialogs instances, which requires an additional proxy as the read entry, and the proxy is missing now. 
Should we just disable deployment mode for HA now, since it only returns partial responses?

Reply:
done